model_cards/tartuNLP/EstBERT/README.md

```diff
@@ -0,0 +1,41 @@
+---
+language: et
+---
+# EstBERT
+
+
+### What's this?
+The EstBERT model is a pretrained BERT<sub>Base</sub> model exclusively trained on Estonian cased corpus on both 128 and 512 sequence length of data. 
+
+### How to use?
+You can use the model transformer library both in tensorflow and pytorch version. 
+```
+from transformers import AutoTokenizer, AutoModelForMaskedLM
+tokenizer = AutoTokenizer.from_pretrained("tartuNLP/EstBERT")
+model = AutoModelForMaskedLM.from_pretrained("tartuNLP/EstBERT")
+```
+You can also download the pretrained model from here, [EstBERT_128]() [EstBERT_512]()
+#### Dataset used to train the model
+The EstBERT model is trained both on 128 and 512 sequence length of data. For training the EstBERT we used the [Estonian National Corpus 2017](https://metashare.ut.ee/repository/browse/estonian-national-corpus-2017/b616ceda30ce11e8a6e4005056b40024880158b577154c01bd3d3fcfc9b762b3/), which was the largest Estonian language corpus available at the time. It consists of four sub-corpora: Estonian Reference Corpus 1990-2008, Estonian Web Corpus 2013, Estonian Web Corpus 2017 and Estonian Wikipedia Corpus 2017.
```

Comment:
If possible you should add dataset ids to your metadata. Even if those are not implemented in `huggingface/datasets` yet, this will prompt us (or the community) to add them at some point.

cc @lhoestq @thomwolf